crates/rpc/rpc-types/src/eth/transaction/receipt.rs

```diff
@@ -9,7 +9,7 @@ pub struct TransactionReceipt {
     /// Transaction Hash.
     pub transaction_hash: Option<H256>,
     /// Index within the block.
-    pub transaction_index: Option<U256>,
+    pub transaction_index: u64,
```

Comment:
this should be U64 because should be hex

Reply:
Gotcha makes sense, thanks! Just changed it to U64 🙂